### ISSUE TITLE
Crash due to high balance: Int cannot represent non 32-bit signed integer value: 25000000000

### ISSUE THREAD

**Issue by Kixunil** [apotdevin/thunderhub#122]
**Describe the problem/bug**

It seems like ThunderHub crashes if the user has too much money. :)
After clicking on `Connect` of SSO account, I get error 500 and this is in logs:

```
Aug 12 18:04:28 disp1676 thunderhub[5134]: { Error: Int cannot represent non 32-bit signed integer value: 25000000000
Aug 12 18:04:28 disp1676 thunderhub[5134]:     at new ApolloError (/usr/lib/thunderhub/node_modules/@apollo/client/errors/errors.cjs.js:31:28)
Aug 12 18:04:28 disp1676 thunderhub[5134]:     at /usr/lib/thunderhub/node_modules/@apollo/client/core/core.cjs.js:1385:47
Aug 12 18:04:28 disp1676 thunderhub[5134]:     at /usr/lib/thunderhub/node_modules/@apollo/client/utilities/utilities.cjs.js:921:69
Aug 12 18:04:28 disp1676 thunderhub[5134]:     at new Promise (<anonymous>)
Aug 12 18:04:28 disp1676 thunderhub[5134]:     at Object.next (/usr/lib/thunderhub/node_modules/@apollo/client/utilities/utilities.cjs.js:921:21)
Aug 12 18:04:28 disp1676 thunderhub[5134]:     at notifySubscription (/usr/lib/thunderhub/node_modules/zen-observable/lib/Observable.js:135:18)
Aug 12 18:04:28 disp1676 thunderhub[5134]:     at onNotify (/usr/lib/thunderhub/node_modules/zen-observable/lib/Observable.js:179:3)
Aug 12 18:04:28 disp1676 thunderhub[5134]:     at SubscriptionObserver.next (/usr/lib/thunderhub/node_modules/zen-observable/lib/Observable.js:235:7)
Aug 12 18:04:28 disp1676 thunderhub[5134]:     at /usr/lib/thunderhub/node_modules/@apollo/client/utilities/utilities.cjs.js:909:68
Aug 12 18:04:28 disp1676 thunderhub[5134]:     at Array.forEach (<anonymous>)
Aug 12 18:04:28 disp1676 thunderhub[5134]:   graphQLErrors:
Aug 12 18:04:28 disp1676 thunderhub[5134]:    [ { GraphQLError: Int cannot represent non 32-bit signed integer value: 25000000000
Aug 12 18:04:28 disp1676 thunderhub[5134]:          at GraphQLScalarType.serializeInt [as serialize] (/usr/lib/thunderhub/node_modules/graphql/type/scalars.js:53:11)
Aug 12 18:04:28 disp1676 thunderhub[5134]:          at completeLeafValue (/usr/lib/thunderhub/node_modules/graphql/execution/execute.js:668:37)
Aug 12 18:04:28 disp1676 thunderhub[5134]:          at completeValue (/usr/lib/thunderhub/node_modules/graphql/execution/execute.js:614:12)
Aug 12 18:04:28 disp1676 thunderhub[5134]:          at /usr/lib/thunderhub/node_modules/graphql/execution/execute.js:527:16
Aug 12 18:04:28 disp1676 thunderhub[5134]:        message:
Aug 12 18:04:28 disp1676 thunderhub[5134]:         'Int cannot represent non 32-bit signed integer value: 25000000000',
Aug 12 18:04:28 disp1676 thunderhub[5134]:        locations: [],
Aug 12 18:04:28 disp1676 thunderhub[5134]:        path: [Array] } ],
Aug 12 18:04:28 disp1676 thunderhub[5134]:   networkError: null,
Aug 12 18:04:28 disp1676 thunderhub[5134]:   message:
Aug 12 18:04:28 disp1676 thunderhub[5134]:    'Int cannot represent non 32-bit signed integer value: 25000000000',
Aug 12 18:04:28 disp1676 thunderhub[5134]:   extraInfo: undefined }
```

```
$ sudo lncli --network regtest  walletbalance
{
    "total_balance": "25000000000",
    "confirmed_balance": "25000000000",
    "unconfirmed_balance": "0"
}
```

So I suppose It's the balance that's causing the crash.

**Your environment**
* Version of ThunderHub: 0.9.3
* Deployment method: Debian package
* Other relevant environment details: Regtest network with high `lnd` balance

**To Reproduce**
1. Install regtest LND
2. Mine 5 blocks to LND address (+100 more to make them buried)
3. Open ThunderHub (with SSO)

**Expected behavior**
No crash, ThunderHub shows the correct balance

**Actual behavior**
ThunderHub crashes because the amount is too high

**Comment by apotdevin** [comment]
Too much money! Good catch, i'll fix it up

**Comment by apotdevin** [comment]
Fixed with 603dcacd027ed74419f4458168d6db8b1260d8ff
